src/doc/contrib/src/architecture/filesystem.md

```diff
@@ -0,0 +1 @@
+# Filesystem
```

Comment:
This file seems empty. Was that intentional?

Reply:
#11873 addresses it